src/cs-core/tools/Mark.cpp

```diff
@@ -74,16 +74,16 @@ void main()
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
 Mark::Mark(std::shared_ptr<InputManager> pInputManager, std::shared_ptr<SolarSystem> pSolarSystem,
-    std::shared_ptr<GraphicsEngine> graphicsEngine, std::shared_ptr<TimeControl> pTimeControl,
+    std::shared_ptr<Settings> settings, std::shared_ptr<TimeControl> pTimeControl,
     std::string const& sCenter, std::string const& sFrame)
     : mInputManager(std::move(pInputManager))
     , mSolarSystem(std::move(pSolarSystem))
-    , mGraphicsEngine(std::move(graphicsEngine))
+    , mSettings(std::move(settings))
     , mTimeControl(std::move(pTimeControl))
-    , mVAO(std::make_unique<VistaVertexArrayObject>())
-    , mVBO(std::make_unique<VistaBufferObject>())
-    , mIBO(std::make_unique<VistaBufferObject>())
-    , mShader(std::make_unique<VistaGLSLShader>()) {
+    , mVAO(new VistaVertexArrayObject())
+    , mVBO(new VistaBufferObject())
+    , mIBO(new VistaBufferObject())
+    , mShader(new VistaGLSLShader()) {
```

Comment:
Why use `new` instead of `make_unique`? Especially since you don't see the type here, one might assume that raw pointers are used.

Reply:
This seems to got lost during the (awful) merge from develop...